### PR TITLE
chore: improve PR template

### PR DESCRIPTION
**Pull request by andrii-bodnar**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
+
+## Types of changes
+
+[//]: # (What types of changes does your code introduce to Lingui?)
+[//]: # (_Put an `x` in the boxes that apply_)
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update
```
Comment (Martin005) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:13]: Maybe it would be good to add "Chore" as a type of change 🙂

Comment (andrii-bodnar) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:13]: Good point 🙂

Comment (andrii-bodnar) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:13]: Or maybe we can leave here only these essential types. Detailed type anyway will be specified in the PR title, it's not required to select these checkboxes in this case (chore, ci, refactoring, etc)

Comment (Martin005) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:13]: I think we can leave only the essential types here for now

**Review by Martin005 — APPROVED** [review]

**Pull request merged**